convert-pth-to-ggml.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
```

Comment:
You can ask for `python3` here if needed, but I feel it's not necessary since the Python 2 sunset was on 2020,

https://www.python.org/doc/sunset-python-2/